front/temporal/hard_delete/client.ts

```diff
@@ -0,0 +1,50 @@
+import type { Result } from "@dust-tt/types";
+import { Err, Ok } from "@dust-tt/types";
+import {
+  ScheduleAlreadyRunning,
+  ScheduleOverlapPolicy,
+} from "@temporalio/client";
+
+import { getTemporalClient } from "@app/lib/temporal";
+import logger from "@app/logger/logger";
+import { getPurgeRunExecutionsScheduleId } from "@app/temporal/hard_delete/utils";
+import { purgeRunExecutionsCronWorkflow } from "@app/temporal/hard_delete/workflows";
+
+import { QUEUE_NAME } from "./config";
+
+/**
+ * This function starts a cron to clean up expired run executions.
+ */
+export async function launchPurgeRunExecutionsSchedule(): Promise<
+  Result<undefined, Error>
+> {
+  const client = await getTemporalClient();
+  const scheduleId = getPurgeRunExecutionsScheduleId();
+
+  try {
+    await client.schedule.create({
+      action: {
+        type: "startWorkflow",
+        workflowType: purgeRunExecutionsCronWorkflow,
+        args: [],
+        taskQueue: QUEUE_NAME,
+      },
+      scheduleId,
+      policies: {
+        catchupWindow: "1 day",
```

Comment:
Why did you pick this config? 🙏🏻 

Our confluence workflow which has some help comments to understand the config, it says: 
```
// If Temporal Server is down or unavailable at the time when a Schedule should take an Action.
// Backfill scheduled action up to 1 previous day.
```

Why do we care about a potential backfilling? If runs are not handled day 1 they will be per default on any of the next call since we check on created < xxxxx. 

Also, why Buffering and not skipping for the overlap config?  🙏🏻

Reply:
The objective was to efficiently manage the initial batch, which involves deleting over 4 million rows. However, I believe further simplification is possible. I have removed the `catchupWindow` and set it to skip.